lib/course_planner/events/events.ex

```diff
@@ -17,15 +17,30 @@ defmodule CoursePlanner.Events do
 
   def all do
     query = from e in Event,
-    order_by: [desc: e.starting_time, desc: e.finishing_time]
+    order_by: [asc: e.date, asc: e.starting_time, asc: e.finishing_time]
```

Comment:
why did you change the ordering to ascending ?

Reply:
I'm not sure, I had the impression that `desc` was wrong.

Reply:
This way the earliest event ever to happen always stay on top regardless of the current date, so after create some events, the user should always scroll down.

I have a few suggestion, but do tell me what do you think
1 - to have three sections : (past and upcoming events) . now if past events are sorted Descending and upcoming is sorted ascending the one on the top is whether the one that lately happen or the one that is going to happen the earliest  
2 - to sort everything descending, this i have applied in other places and solve the problem for a while
3 - to write a generic sort solution for events, task, terms and classes. mainly for all entities that have time or date or both

Reply:
We can finish this PR with **2** and discuss **3**